Dockerfile

```diff
@@ -1,14 +1,15 @@
-FROM alpine:3.4
+FROM alpine:3.6
 
-# The official Promtheus base image has no package manager so rather than
+# The official Prometheus base image has no package manager so rather than
 # artisanally hand-rolling curl and the rest of our stack we'll just use
 # Alpine so we can use `docker build`.
 
 RUN apk add --update curl
 
 # add Prometheus. alas, the Prometheus developers provide no checksum
-RUN export prom=prometheus-1.3.0.linux-amd64 \
-    && curl -Lso /tmp/${prom}.tar.gz https://github.com/prometheus/prometheus/releases/download/v1.3.0/${prom}.tar.gz \
+RUN export PROM_VERSION=1.7.1 \
+    && export prom=prometheus-${PROM_VERSION}.linux-amd64 \
+    && curl -Lso /tmp/${prom}.tar.gz https://github.com/prometheus/prometheus/releases/download/v${PROM_VERSION}/${prom}.tar.gz \
```

Comment:
I'd probably download this archive and manually create a `sha256sum` checksum just to be thorough. You can follow the same pattern as Consul below this.